server/routes/index.js

```diff
@@ -1,6 +1,14 @@
+require("dotenv").config();
 const express = require("express");
 const router = express.Router();
 
+const { loginRouter } = require("./login.router");
+const { signup } = require("./signup.router");
```

Comment:
For consistently, the exports from these two files should probably share the same naming convention (`loginRouter` and `signupRouter`, for example).

Reply:
Got it and have it modified already.